<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UAlbertaControlWidget</class>
 <widget class="QWidget" name="UAlbertaControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>1292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Helicopter Status</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout_6" columnstretch="0,0,0,0">
          <item row="0" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Rotor RPM:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="rotor_rpm_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="engine_rpm_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="2" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Engine RPM:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="receiver_bat_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="avionics_bat_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="2" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_20">
            <property name="toolTip">
             <string>Receiver battery status</string>
            </property>
            <property name="text">
             <string>Receiver Bat:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_15">
            <property name="toolTip">
             <string>Avionics battery level</string>
            </property>
            <property name="text">
             <string>Avionics Bat:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="5" column="3">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="horizontalGroupBox">
       <property name="title">
        <string>GPS Status</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QGridLayout" name="gridLayout_11">
          <item row="1" column="2" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_23">
            <property name="toolTip">
             <string>Position Standard Deviation</string>
            </property>
            <property name="text">
             <string>Position σ:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_22">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Pos. Sol. Status:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_25">
            <property name="toolTip">
             <string>Position solution type</string>
            </property>
            <property name="text">
             <string>Pos. Sol. Type:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_30">
            <property name="toolTip">
             <string>Velocity Solution Type</string>
            </property>
            <property name="text">
             <string>Vel. Sol. Type:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="pos_sol_type_label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="satellites_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="vel_sol_type_label">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="pos_sol_status_label">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="2" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_31">
            <property name="toolTip">
             <string>Velocity Standard Deviation</string>
            </property>
            <property name="text">
             <string>Velocity σ:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="vel_std_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="pos_std_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="2" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_32">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>The number of satellites that make up this solution</string>
            </property>
            <property name="text">
             <string>Satellites:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <spacer name="horizontalSpacer_10">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="3">
           <spacer name="horizontalSpacer_11">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>Flight Information</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,1,0">
          <item row="4" column="1">
           <widget class="QLabel" name="nav_euler_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Nav Filter Message:</string>
            </property>
           </widget>
          </item>
          <item row="11" column="1">
           <widget class="QLabel" name="collective_pitch_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="gx3_message_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="10" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>AHRS Euler (deg):</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QLabel" name="position_reference_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="gx3_state_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_3">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">text-align:right;</string>
            </property>
            <property name="text">
             <string>Nav Filter State:</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QLabel" name="ned_origin_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="8" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>NED Origin (LLH):</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLabel" name="ned_velocity_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="4" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Nav Euler (deg):</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLabel" name="llh_position_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="5" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>LLH Position:</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>Position Setpoint (NED)</string>
            </property>
           </widget>
          </item>
          <item row="8" column="2">
           <widget class="QPushButton" name="ned_origin_button">
            <property name="text">
             <string>Set Origin</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QLabel" name="ahrs_euler_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="11" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_28">
            <property name="text">
             <string>Collective Pitch (deg):</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>NED Velocity:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLabel" name="ned_position_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="6" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>NED Position:</string>
            </property>
           </widget>
          </item>
          <item row="12" column="0">
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>Alitimeter Height (m):</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="12" column="1">
           <widget class="QLabel" name="altimeter_height_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="set_attitude_button">
            <property name="text">
             <string>Init Filter</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="reset_filter_button">
            <property name="text">
             <string>Reset Filter</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Control Status</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Servo Source:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="servo_source_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Pilot Mode:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="pilot_mode_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Control Mode:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="control_mode_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Attitude Source:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="attitude_source_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="1">
        <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1">
         <item row="0" column="1">
          <widget class="QComboBox" name="servo_source_box"/>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Set Servo Source:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Set Control Mode:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Set Attitude Source:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="servo_source_button">
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="control_mode_button">
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="attitude_source_button">
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="attitude_source_box"/>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="control_mode_box"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="setRefPos">
         <property name="focusPolicy">
          <enum>Qt::WheelFocus</enum>
         </property>
         <property name="text">
          <string>Set Reference Position</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="shutdown_button">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(255, 0, 4)</string>
         </property>
         <property name="text">
          <string>Kill Autopilot</string>
         </property>
         <property name="icon">
          <iconset resource="../../../qgroundcontrol.qrc">
           <normaloff>:/images/actions/process-stop.svg</normaloff>:/images/actions/process-stop.svg</iconset>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="new_log_point_button">
       <property name="text">
        <string>Start New Log Point</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../qgroundcontrol.qrc"/>
 </resources>
 <connections/>
</ui>
